workers/unity/Packages/com.improbable.gdk.playerlifecycle/PlayerLifecycleHelper.cs

```diff
@@ -21,6 +23,47 @@ public static class PlayerLifecycleHelper
             template.AddComponent(owningComponent, serverAccess);
         }
 
+        public static bool SerializeArguments(Object playerCreationArguments, out byte[] serializedArguments)
```

Comment:
We are we providing a helper for this? What if the user wants to control their own serialisation? Also note that our types are not marked as serializable as we expect that to happen via schema objects.

Reply:
Our types are marked as serializable since last week.

Reply:
Martijn merged in a change to make types, enums and snapshots serializable